mxcubeweb/core/models/configmodels.py

```diff
@@ -19,7 +19,7 @@ class FlaskConfigModel(BaseModel):
     ALLOWED_CORS_ORIGINS: list[str] = Field([], description="")
     SECURITY_PASSWORD_SALT: str = Field("ASALT", description="")
     SECURITY_TRACKABLE: bool = Field(True, description="")
-    USER_DB_PATH: str = Field("/tmp/mxcube-user.db", description="")
+    USER_DB_PATH: str = Field("", description="")
```

Comment:
Why the empty strings? I am really asking, I have not used Pydantic in a while, I do not remember how it works.

Isn't it enough like this?

```suggestion
    USER_DB_PATH: str
```

Reply:
we need to declare that it's a Field instance for Pydantic even with nothing ("").
If we let just **str** it's just a type hint with no value and it became a required field, 
Basically the first one has a default value and the second does not.
Anyway that was the first solution but it will be changed.

Reply:
> we need to declare that it's a Field instance for Pydantic even with nothing ("").
> If we let just **str** it's just a type hint with no value and it became a required field,
> Basically the first one has a default value and the second does not.

Ah right, thanks for the refresher. Indeed, I think that we want it to be optional, not required, with the default value hard-coded to `/tmp/mxcube-user.db` then, right? The empty string does not make sense to me.